src/player_activity.h

```diff
@@ -133,6 +137,36 @@ class player_activity : public JsonSerializer, public JsonDeserializer
          * Returns true if the activity is complete.
          */
         bool is_complete() const;
+
+        activity_type type = ACT_NULL; /** type of activity. */
```

Comment:
Because I was working on `player_activity` at the time and a number of the members were undocumented and could be partially encapsulated which allows some assumptions to be made about where some members can be set. I can change to comments on the leading line but I'm not sure if that style change is documented so it might be worth drawing everyone's attention to that.